js/background.js

```diff
@@ -28,8 +28,11 @@ function Background() {
 
   localStorage['os'] = os;
 
-  if (localStorage['prev_version'] === undefined) {
-    localStorage['prev_version'] = chrome.runtime.getManifest().version;
+  chrome.runtime.onInstalled.addListener(function(details) {
+    // only run the following section on install
+    if (details.reason !== "install") {
```

Comment:
@bsstoner this `reason` part is better described here: https://developer.chrome.com/extensions/runtime#type-OnInstalledReason